src/Components/NotFound/NotFound.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import './NotFound.css';
+
+const NotFound = () => {
+  return (
+    <section className='notfound-container'>
+      <h1 className='notfound-header'>Oh no! This page either moved or doesn't exist. Try something else.</h1>
+      <div className='notfound-blurb'>
+        <p>PS - Hope you're doing well. 🤍</p>
+        <p>PPS - Proud of you.</p>
+      </div>
+    </section>
```

Comment:
this is super cute, pulled it down just to see how cute and confirmed, its rad!

Reply:
Love that supportive error handling

Reply:
so sweet, I love it